scripts/winesapos-read-only.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+mount -o remount,ro /
```

Comment:
Simple and effective! I still want to go with a different solution long-term that will help with day 2 operations of upgrades and keeping consistency. For now, users could potentially enable this read-only service if they want it, disable it to run a winesapOS Upgrade, and then re-enable it again.

I don't think the first-time setup and other applications will properly work in this mode but we can add it for now and users/developers can test it out.

Reply:
maybe make a desktop shortcut which activates / disables the `read-only` service

Reply:
https://github.com/GuestSneezeOSDev/winesapOS/blob/main/files/winesap-read-only.service

Reply:
Good idea. I'll try that out.